test/testenv/secretutil.go

```diff
@@ -144,22 +154,27 @@ func ModifySecretKey(deployment *Deployment, ns string, key string, value string
 }
 
 // UpdateSecret Updates the secret object based on SecretResponse Struct
-func UpdateSecret(deployment *Deployment, ns string, secretObj SecretResponse) (bool, error) {
+func UpdateSecret(deployment *Deployment, ns string, secretObj SecretResponse, delete bool) error {
```

Comment:
I think `UpdateSecretObject` makes more sense?

Reply:
The update CR requires the data in map[string]byte format. Calling the function directly from the Test case would involve. An option would be to break the functions and run the data modification in the function itself

Reply:
Merged the 2 functions into 1 as it was just creating confusion